sdk/storage/Azure.Storage.Blobs/tests/BlockBlobClientTests.cs

```diff
@@ -2788,7 +2788,7 @@ public async Task SyncUploadFromUriAsync_Error()
 
         [RecordedTest]
         [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2020_04_08)]
-        public async Task SyncUploadFromUriAsync_OverwiteSourceBlobProperties()
+        public async Task SyncUploadFromUriAsync_overwriteSourceBlobProperties()
```

Comment:
TODO: renaming this back to correct camel case. I think this got caught in my mass rename of overwite to overwrite